CONTRIBUTING.md

```diff
@@ -19,30 +19,82 @@ you maintain a collaborative and inclusive environment for everyone involved.
-    - Provide a clear and detailed description of the changes you made in the PR.
-    - Link any relevant issues or discussions in the PR description.
-    - Pull Request Headers should follow the below `Naming Conventions of Pull Requests` hader in [README.md](https://github.com/afet-yonetim-sistemi/ays-be/edit/main/README.md) file.
-
-Once your PR is submitted, the project maintainers will review it and provide feedback. Be prepared to make additional
-changes based on the feedback received.
+Thank you for considering contributing to our project! To make the process smooth, please follow these steps:
+
+1. **Fork the Repository:**
+   - Visit our [GitHub repository](https://github.com/afet-yonetim-sistemi/ays-be) and fork it to your GitHub account.
+
+2. **Create a New Branch:**
+   - Create a branch for your contribution by referring to
+     the [Naming Conventions of Branches](#branch-naming-conventions) for guidance.
+   - Ensure your feature branch is associated with the corresponding GitHub issue from the 'Development' section in
+     each issue page.
+
+3. **Make Changes:**
+   - Implement your changes in the branch you created.
+   - Thoroughly test your changes to ensure they work as expected.
+
+4. **Commit Changes:**
+   - Commit your changes with clear and concise messages. Refer to the [Commit Messages](#commit-messages) section for
+     guidance.
+
+5. **Push Changes:**
+   - Push your changes to your forked repository.
+
+6. **Open a Pull Request (PR):**
+   - Open a PR against the `main` branch of the original repository by referring to
+     the [Pull Request Naming Conventions](#pull-request-naming-conventions) for guidance.
+   - Provide a detailed description of your changes in the PR.
+   - Link relevant issues or discussions in the PR description.
+   - Follow the pull request naming conventions outlined below.
+   - PR title should contain the issue/feature/bug number.
+
+7. **Review and Feedback:**
+   - Once your PR is submitted, project maintainers will review it and provide feedback.
+   - Be responsive to feedback and make necessary changes.
+
+## Development Standards
+
+Our project adheres to the [GitHub flow](https://docs.github.com/en/get-started/quickstart/github-flow) for
+collaboration.
+Issues are managed on GitHub Discussions, with plans to transition them to Jira.
+
+### Branch Naming Conventions:
+
+- `bugfix/{jira-issue-number}/{optional-description}`
+- `feature/{jira-issue-number}/{optional-description}`
+- `refactor/{jira-issue-number}/{optional-description}`
+- `hotfix/{jira-issue-number}/{optional-description}`
+
+> Example : `feature/AYS-1/add-feature-to-handle-user-authentication`
+
+### Pull Request Naming Conventions:
+
+- `#{jira-issue-number} | {header-for-summary-of-development}`
+
+> Example : `#AYS-1 | Add feature to handle user authentication`
+
+Here are some additional standards to keep in mind:
+
+- Link your feature branch to the corresponding GitHub issue.
```

Comment:
Şu kısmı şöyle yapabilirsek daha iyi olur diye düşünüyorum. @agitrubard 

**Linking Feature Branch to GitHub Issue:**
- Ensure your feature branch is linked to the corresponding GitHub issue for effective tracking.
**Protected main Branch:**
- The main branch is protected. All changes must go through pull requests.
**Linking Pull Requests to GitHub Project:**
- Associate your pull request with the relevant GitHub issue within the project board.
**Single Code Owner's Approval:**
- Only one code owner's approval is needed for merging feature branches.
**Squash-Merged Pull Requests:**
- Squash-merge your pull requests to maintain a clean commit history.
**Build and Test Verification:**
- Ensure your pull request passes the build and necessary tests before merging.
**Version Tagging for Releases:**
- Tag releases with an appropriate version number for tracking and deployment.
**Subjectless Commit Messages:**
- Write subjectless commit messages for clear and consistent communication.
**Timely Resolution of Comments:**
- Resolve comments within the pull request promptly.
**Conciseness and Single Focus:**
- Keep pull requests concise and address one task or issue at a time.
**Avoiding Secret/Credential Information:**
- Refrain from including secret or credential information in your contributions.
**Adherence to Coding Standards:**
- Code should comply with existing coding standards; introducing new standards requires justification.

Reply:
GitHub issue kullanmıyoruz artık, bu yüzden GPT'ye Jira'ya göre yazdırabilirsen çok iyi olabilir 🙌🏼